cargo/Cargo.toml

```diff
@@ -29,7 +29,15 @@ embassy = ["esp-idf-svc/embassy-sync", "esp-idf-svc/critical-section", "esp-idf-
 
 [dependencies]
 log = { version = "0.4", default-features = false }
-esp-idf-svc = { version = "0.48", default-features = false }
+# TODO: esp-idf-svc = { version = "0.49", default-features = false }
```

Comment:
Let's just update to the new versions. `esp-idf-svc` will be released in a few minutes.